File: src/main/java/com/projectswg/forwarder/services/client/ClientConnectionManager.java
package com.projectswg.forwarder.services.client;

import me.joshlarson.jlcommon.control.Manager;
import me.joshlarson.jlcommon.control.ManagerStructure;

@ManagerStructure(children = {
		ClientInboundDataService.class,
		ClientOutboundDataService.class,
		ClientProtocolService.class,
		ClientServerService.class
})
public class ClientConnectionManager extends Manager {
	
}

File: src/main/java/com/projectswg/forwarder/services/client/ClientConnectionManager.kt
package com.projectswg.forwarder.services.client

import me.joshlarson.jlcommon.control.Manager
import me.joshlarson.jlcommon.control.ManagerStructure

@ManagerStructure(children = [
	ClientInboundDataService::class,
	ClientOutboundDataService::class,
	ClientProtocolService::class,
	ClientServerService::class
])
class ClientConnectionManager : Manager()
